packages/jaeger-ui/src/model/ddg/types.tsx

```diff
@@ -50,3 +51,24 @@ export type TDdgModel = {
   services: TDdgServiceMap;
   visibilityIdxToPathElem: TDdgVisibilityIdxToPathElem;
 };
+
+export type TDdgEdgeIdentifiers = 'egressEdges' | 'ingressEdges';
+
+export class DdgVertex {
```

Comment:
This type should be in own file with test, and should be compatible with plexus vertex type